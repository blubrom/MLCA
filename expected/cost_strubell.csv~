Models,Hours,Expected energy,Estimated energy,Expected electricity cost,Estimated electricity cost
,,(kWh),(kWh),($),($)
1,120,41.7,93,5,11
24,2880,983,2230,118,267
4789,239942,82250,186000,9870,22320