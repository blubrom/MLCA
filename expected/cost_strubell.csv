Models,Hours,Expected,Estimated,Expected,Estimated
,,energy,energy,electricity,electricity
,,(kWh),(kWh),cost ($),cost ($)
1,120,41.7,93,5,11
24,2880,983,2230,118,267
4789,239942,82250,186000,9870,22320